File: 344-ReverseString/344-ReverseString.cpp
// Last updated: 3/16/2026, 8:23:01 AM
class Solution {
public:
    void reverseString(vector<char>& s) {
        int left = 0, right = s.size() - 1;
        while(left<right){
            char temp = s[left];
            s[left] = s[right];
            s[right] = temp;
            left++;
            right--;
        }
    }
};

File: 344-ReverseString/344-ReverseString.java
// Last updated: 3/16/2026, 8:22:58 AM
class Solution {
    public void reverseString(char[] arr) {
        int left = 0, right = arr.length - 1;
        while(left < right){
            char temp = arr[left];
            arr[left] = arr[right];
            arr[right] = temp;
            left++;
            right--;
        }
    }
}